docs/source/losses.rst

```diff
@@ -137,3 +137,13 @@ Loss Wrappers
 ~~~~~~~~~~~~~~~~~~~~~
 .. autoclass:: DeepSupervisionLoss
     :members:
+
+`SoftclDiceLoss`
+~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Thanks for the PR.
Here should be aligned with the class name and also line 147.
Also, these two losses should be moved to the `Segmentation Losses` part.
https://github.com/Project-MONAI/MONAI/blob/eba3b52acc29054ae8ea29d64674ead132602f5b/docs/source/losses.rst#segmentation-losses

Reply:
Hi @KumoLiu, Thank you for your help. I have completed the correction and please help me review it again.
Thanks!